packages/react-components/src/components/LocalVideoTile.tsx

```diff
@@ -5,6 +5,8 @@ import { Stack } from '@fluentui/react';
 import { _formatString } from '@internal/acs-ui-common';
 import React, { useMemo } from 'react';
 import { OnRenderAvatarCallback, VideoStreamOptions, CreateVideoStreamViewResult } from '../types';
+/* @conditional-compile-remove(reaction) */
+import { Reaction } from '../types';
```

Comment:
We should be getting this type from the stateful client package off the callState. We don't want to have multiple definitions of the same type throughout the packages.

Reply:
Update after our verbal discussion: This type is from the react-component's VideoTile's prop type definition which is different from the stateful client's state object.